### DISCUSSION TITLE
Profile过多问题

### DISCUSSION THREAD

**Discussion by Accelarator** [apache/doris#14189] [A - General / Q&A]
profile太多，使用show query profile "/"\G经常加载出一大堆，请问有什么参数可以过滤出前几条，或者删除掉旧的profile吗

**Comment by LemonLiTree** [comment]
Profile只保留最近的100条，会自动删除，目前不支持手动删除。

**Reply by Accelarator** [reply]
可以类似grep一样过滤出前3条之类的吗

**Comment by dataalive** [comment]
如果平常不需要的话，可以平常关掉。`set global enable_profile = false`
